src/libs/libRocket/Include/Rocket/Core/StringBase.inl

```diff
@@ -149,6 +150,12 @@ const T* StringBase< T >::CString() const
 }
 
 template< typename T >
+bool StringBase<T>::_IsUsingLocalBuffer() const
```

Comment:
libRocket doesn't use the underscore in front of the function anywhere else. Just leave it as Is